pkg/cloudprovider/openstack_test.go

```diff
@@ -409,8 +410,19 @@ func portListHandler(t *testing.T, w http.ResponseWriter, r *http.Request) {
 	deviceOwner := r.URL.Query().Get("device_owner")
 	networkID := r.URL.Query().Get("network_id")
 
+	// The "undefined" test case in TestGetNodeEgressIPConfiguration relies on stable sort order of the returned neutron
+	// interfaces. Let's make our lives easier and let's suppose that neutron always returns ports in stable order for
```

Comment:
What is neutrons behavior in general? does it return ports in a stable order?
I am fine with the approach you took but an alternative would be to expand `expectedNodeEgressIPConfig` to allow for more than one return possibility.

Reply:
ok I expanded expectedNodeEgressIPConfig instead  - it doesn't really make a difference to me